cmd/cerca/main.go

```diff
@@ -106,6 +108,10 @@ func run() {
 		complain(fmt.Sprintf("couldn't create dir '%s'", dataDir))
 	}
 	config := util.ReadConfig(configPath)
+	_, err = util.CreateIfNotExist(filepath.Join("html", "assets", "theme.css"), defaults.DEFAULT_THEME)
```

Comment:
@cblgh ah, on the upgrade, this caught me out with a recurring:

```
couldn't output default theme.css
```

i guess this has something to do with the path that is being constructed? on my remote install, i run `cerca` via a systemd conf file and it can't create a `html/assets/...` from where it is at `/usr/local/bin/cerca` 😬

i would assume a more local/remote install fix here would be using sort of general "data dir" that can be used by all the commands that need a "root" path for the install. right now, we just have a config option for a path that the database uses. maybe the basepath of the `-config` option?

this could be generalized? im gonna revert this on my local to get the upgrade working, so no rush. hope this makes sense

Reply:
@decentral1se oh no! can you give me more info about how you're concretely invoking cerca on the permacomputing server?

that would give me some baseline to use for the better solution wrt a configurable data dir! i'm just using binaries in a dir similar to the repo structure, so i don't have the full picture

Reply:
@cblgh not to worry, it's been patched / updated for now.

Docs are somewhat here https://permacomputing.net/eik/ and also using https://github.com/cblgh/cerca/blob/main/docs/hosting.md. You can basically see how it's invoked via the systemd unit file. We have a binary somewhere on the system and we point it to `/var/www/bbs...`. So, the repository does not come along for the ride!